presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -620,6 +621,11 @@ public SystemSessionProperties(
                         MAX_CONCURRENT_MATERIALIZATIONS,
                         "Maximum number of materializing plan sections that can run concurrently",
                         featuresConfig.getMaxConcurrentMaterializations(),
+                        false),
+                booleanProperty(
+                        PUSHDOWN_SUBFIELDS_ENABLED,
+                        "Experimental: enable subfield pruning",
+                        false,
```

Comment:
nit: It might be worth adding a configuration property as well for completeness.